src/annotator/test/integration/multi-frame-test.js

```diff
@@ -1,19 +1,18 @@
 import { DEBOUNCE_WAIT } from '../../frame-observer';
-import { CrossFrame, $imports } from '../../cross-frame';
+import { HypothesisInjector } from '../../hypothesis-injector';
 import { isLoaded } from '../../util/frame-util';
 
-describe('CrossFrame multi-frame scenario', () => {
-  let fakeAnnotationSync;
-  let fakeBridge;
-  let proxyAnnotationSync;
-  let proxyBridge;
+describe('HypothesisInjector-FrameObserver-frameUtil integration', () => {
```

Comment:
Naming integration tests after all the components involved is liable to get out of date after future refactoring. I would probably either use a description which refers to the overall process that is being tested (eg. "Anchoring integration test", "Threading integration test") or the main class that serves as the entry point for the functionality being tested (eg. "HypothesisInjector integration test")